app/main.py

```diff
@@ -1,8 +1,42 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
```

Comment:
better use this formatting, fix throughout code
```suggestion
    def __init__(
            self, 
            comfort_class: int,
            clean_mark: int, 
            brand: str,
    ) -> None:
```